code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1028,6 +1028,292 @@ var/global/list/obj/item/device/pda/PDAs = list()
 							dat += {"<li>[mkr.name] ([mkr.x]/[mkr.y]) <a href='byond://?src=\ref[src];choice=removeMarker;rMark=[mkr.num]'>remove</a></li>"}
 						dat += {"</ul>"}
 
+			if (105)//Snake II

```

Comment:
Then at least make the apps you're adding defines.

I'm honestly disappointed the first apps you added are magic-number'd as well.